proto/planning/builder.proto

```diff
@@ -34,11 +34,14 @@ message StartBuildRequest {
     // Data used to construct a sample-based PRM. The resulting discrete edges
     // will be used to seed the IRIS regions.
     RoadmapData roadmap_data = 5;
+  // start and goal system confs for problem to solve and insert into db 
+  SystemConf start_sysconf = 6;
+  SystemConf goal_sysconf = 7;
   }
 }
 
-// Response which contains the result of attempting to start computation of a
-// solution for the corresponding StartBuildRequest.
+/** Response which contains the result of attempting to start computation of a
+ * solution for the corresponding StartBuildFromConfsRequest. */
```

Comment:
I commented this on the `planning_service` PR, but I think we'd probably want to break out the "update roadmap" call into its own RPC. An example could be something like:
```
message UpdateRoadmapRequest {
  // Definitions of motion plans to be solved.
  repeated ProblemDef pdef = 1;
  // Options passed to the solver.
  UpdateRoadmapOptions options = 2;
}

message UpdateRoadmapResponse {
  // Metrics of the resultant solution.
  SolutionMetrics metrics = 1;
}
// Update the roadmap by solving a set of motion problems and saving their results.
rpc UpdateRoadmap(UpdateRoadmapRequest) 
  returns (UpdateRoadmapResponse) {}
```